[2024-07-17T09:26:57.962+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-17T09:26:58.063+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Postgres_dag.delete_data_from_table scheduled__2024-07-15T00:00:00+00:00 [queued]>
[2024-07-17T09:26:58.090+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Postgres_dag.delete_data_from_table scheduled__2024-07-15T00:00:00+00:00 [queued]>
[2024-07-17T09:26:58.091+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-17T09:26:58.159+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2024-07-15 00:00:00+00:00
[2024-07-17T09:26:58.176+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3131) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-17T09:26:58.180+0000] {standard_task_runner.py:63} INFO - Started process 3140 to run task
[2024-07-17T09:26:58.179+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Postgres_dag', 'delete_data_from_table', 'scheduled__2024-07-15T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmpzd0abxsm']
[2024-07-17T09:26:58.187+0000] {standard_task_runner.py:91} INFO - Job 41: Subtask delete_data_from_table
[2024-07-17T09:26:58.392+0000] {task_command.py:426} INFO - Running <TaskInstance: Postgres_dag.delete_data_from_table scheduled__2024-07-15T00:00:00+00:00 [running]> on host 67c97fe0936f
[2024-07-17T09:26:58.668+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rayzo' AIRFLOW_CTX_DAG_ID='Postgres_dag' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-15T00:00:00+00:00'
[2024-07-17T09:26:58.670+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-17T09:26:58.706+0000] {sql.py:276} INFO - Executing:  
        delete from drag_runs where dt = '2024-07-15' and dag_id = 'Postgres_dag'
        
[2024-07-17T09:26:58.726+0000] {base.py:84} INFO - Using connection ID 'Postgres_con' for task execution.
[2024-07-17T09:26:58.744+0000] {base.py:84} INFO - Using connection ID 'Postgres_con' for task execution.
[2024-07-17T09:26:58.757+0000] {sql.py:487} INFO - Running statement:  
        delete from drag_runs where dt = '2024-07-15' and dag_id = 'Postgres_dag'
        , parameters: None
[2024-07-17T09:26:58.762+0000] {sql.py:496} INFO - Rows affected: 0
[2024-07-17T09:26:58.765+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-17T09:26:58.777+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Postgres_dag, task_id=delete_data_from_table, run_id=scheduled__2024-07-15T00:00:00+00:00, execution_date=20240715T000000, start_date=20240717T092658, end_date=20240717T092658
[2024-07-17T09:26:58.844+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-17T09:26:58.854+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
